packages/nextjs/components/Footer.tsx

```diff
@@ -58,17 +58,17 @@ export const Footer = () => {
               </p>
               <a
                 className="flex justify-center items-center gap-1"
-                href="https://buidlguidl.com/"
+                href={projectConfig.creatorLink}
                 target="_blank"
                 rel="noreferrer"
               >
-                <BuidlGuidlLogo className="w-3 h-5 pb-1" />
-                <span className="link">BuidlGuidl</span>
+                <projectConfig.creatorSvg className="w-3 h-5 pb-1" />
```

Comment:
Same for svg and creator. Also, svg could be different and wont work well with common styles